packages/mask/src/plugins/Tips/SNSAdaptor/bodyViews/Wallets.tsx

```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles()((theme) => ({
 interface WalletsPageProp {
     wallets: BindingProof[]
     releaseLoading: boolean
-    onRelease: (wallet?: BindingProof) => Promise<boolean>
+    onRelease: (wallet?: BindingProof) => Promise<void>
```

Comment:
```suggestion
    onRelease(wallet?: BindingProof): void | Promise<void>
```

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks

Reply:
done